gse-util/src/main/java/com/powsybl/gse/util/GroovyCodeEditor.java

```diff
@@ -67,10 +80,24 @@
 
     private static final ServiceLoaderCache<KeywordsProvider> KEYWORDS_LOADER = new ServiceLoaderCache<>(KeywordsProvider.class);
 
+    private static final ServiceLoaderCache<AutoCompletionWordsProvider> AUTO_COMPLETION_WORDS_LOADER = new ServiceLoaderCache<>(AutoCompletionWordsProvider.class);
```

Comment:
Why is it both cached in ModificationScriptEditor and GroovyCodeEditor?

Reply:
updated